admin_guide/cluster_capacity.adoc

```diff
@@ -0,0 +1,184 @@
+[[admin-guide-cluster-capacity]]
+= Cluster Capacity Analysis
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Monitoring available resources in the cluster is very important as operators can increase
+the current resources in time before all of them get exhausted, or can carry different
+steps that lead to increase of available resources. Cluster capacity consists of capacities
+of individual cluster nodes. Capacity covers CPU, memory, disk space and other resources.
+
+Overall remaining allocatable capacity is a rough estimation since it does not assume all
+resources being distributed among nodes. Goal is to analyze remaining allocatable resources
+and estimate available capacity that is still consumable in terms of a number of instances
+of a pod with given requirements that can be scheduled in a cluster.
+
+== Running cluster capacity analysis
+
+Cluster capacity analysis tool can be run either as a stand-alone utility from command line,
+or in a pod inside a openshift cluster.
+
+[admin-guide-cluster-capacity-command-line]
+### Running cluster capacity from command line
+
+To run the tool on command line:
+
+```shell
+$ cluster-capacity --kubeconfig <path-to-kubeconfig> --podspec <path-to-pod-spec>
+```
+
+`--kubeconfig` is path to kubeconfig file to cluster and `--podspec` is path to sample pod spec
+that is used by the tool for estimation. The `podspec` specifies its resource requirements as
+`limits` or `requests`. The cluster capacity tool takes the pod's resource requirements into account
+for its estimation analysis. An example pod spec is give below:
+
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: small-pod
+  labels:
+    app: guestbook
+    tier: frontend
+spec:
+  containers:
+  - name: php-redis
+    image: gcr.io/google-samples/gb-frontend:v4
+    imagePullPolicy: Always
+    resources:
+      limits:
+        cpu: 150m
+        memory: 100Mi
+      requests:
+        cpu: 150m
+        memory: 100Mi
+```
+
+
+The output of the cluster capacity command is a number of estimated pods that can be scheduled on the cluster.
+`--verbose` flag can also be provided as part of the above command to output detailed description
+about how many pods can be scheduled on each node in the cluster. An example of verbose output on
+a 2-node cluster is as follows:
+
+```
+small-pod pod requirements:
+	- CPU: 150m
+	- Memory: 100Mi
+
+The cluster can schedule 52 instance(s) of the pod small-pod.
+
+Termination reason: Unschedulable: No nodes are available that match all of the following predicates:: Insufficient cpu (2).
+
+Pod distribution among nodes:
+small-pod
+	- 192.168.124.214: 26 instance(s)
+	- 192.168.124.120: 26 instance(s)
+
+```
+
+[admin-guide-cluster-capacity-pod]
+### Running cluster capacity in a pod
+
+Below is an example of a pod spec (`cluster-capacity-pod.yaml`) that runs cluster capacity image as a container. Note that
```

Comment:
Yeah sure, I will look into that. I thought this documentation was just "a" way to explain how cluster capacity can be used, because most admin would use it in different ways whatever way they want.